src/lib/pages/publish-module/index.tsx

```diff
@@ -192,89 +163,17 @@ export const PublishModule = () => {
-                      fontWeight={700}
-                      color="text.main"
-                      textAlign="start"
-                    >
-                      What is republishing module?
-                    </Text>
-                    <AccordionIcon color="gray.600" ml="auto" />
-                  </AccordionButton>
-                  <AccordionPanel
-                    bg="transparent"
-                    py={3}
-                    px={0}
-                    borderTop="1px solid"
-                    borderColor="gray.700"
-                  >
-                    <Text variant="body2" color="text.dark" p={1}>
-                      In {chainPrettyName}, You can republish the module which
-                      serve the purpose to migrate or upgrade the published
-                      module by uploading new .mv file with similar
-                      configurations. Each policy will provide different
-                      flexibility for further upgrades whether you can add new
-                      functions without maintaining old functions (Arbitrary),
-                      or required to maintain old functions (Compatible).
-                      Choosing “Immutable” will not allow you to make any
-                      changes with this module ever. You should read more about
-                    </Text>
-                    <Flex
-                      align="center"
-                      cursor="pointer"
-                      borderRadius={4}
-                      p={1}
-                      gap={1}
-                      alignItems="center"
-                      width="fit-content"
-                      transition="all 0.25s ease-in-out"
-                      color="secondary.main"
-                      _hover={{
-                        color: "secondary.light",
-                        bgColor: "secondary.background",
-                      }}
-                      // onClick={onClick}
-                    >
-                      <Text
-                        variant="body3"
-                        color="secondary.main"
-                        fontWeight={700}
-                      >
-                        See Initia Doc
-                      </Text>
-                      <CustomIcon
-                        name="chevron-right"
-                        color="secondary.main"
-                        boxSize={3}
-                        m={0}
-                      />
-                    </Flex>
-                  </AccordionPanel>
-                </AccordionItem>
-              </Accordion>
-            </Box>
+          <Box gridArea="2 / 3" pl="32px">
+            <UploadAccordion />
           </Box>
-          <Box w="10%" />
-        </Flex>
-      </Box>
+          <Box gridArea="3 / 3" pl="32px">
+            <PolicyAccordion chainName={chainPrettyName} />
+          </Box>
+        </Grid>
+      </PageContainer>
       <Footer isLoading={false} fieldAmount={fields.length} />
```

Comment:
remove `isLoading={false}` as `false` is already the default value

Reply:
can ignore, will wireup in the next pr